ceph.spec.in

```diff
@@ -887,6 +887,9 @@ Provides:	ceph-test:/usr/bin/ceph-osdomap-tool
 Requires:	ceph-base = %{_epoch_prefix}%{version}-%{release}
 Requires:	sudo
 Requires:	libstoragemgmt
+%if 0%{with seastar}
+Requires:	protobuf
+%endif
```

Comment:
`rpmbuild` will automatically add a `libprotobuf.so.25()(64bit)` dependency to the packages that need it (eg `ceph-osd`), so we don't technically need this `Requires` line in `ceph.spec.in`. It's harmless, but we could remove it.